From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 19:14:18 +0200
Subject: Debianize crm_mon.service

---
 tools/crm_mon.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/crm_mon.service.in b/tools/crm_mon.service.in
index 0c7e8f1..d9976d8 100644
--- a/tools/crm_mon.service.in
+++ b/tools/crm_mon.service.in
@@ -3,7 +3,7 @@ Description=Daemon for pacemaker monitor
 
 [Service]
 Type=forking
-EnvironmentFile=-@sysconfdir@/sysconfig/crm_mon
+EnvironmentFile=-@sysconfdir@/default/crm_mon
 ExecStart=@sbindir@/crm_mon $OPTIONS
 Restart=always
 
