From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 10 Feb 2016 23:26:30 +0100
Subject: Fix typo: Initalizing -> Initializing

---
 crmd/lrm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/crmd/lrm.c b/crmd/lrm.c
index a73b2a9..b9d020f 100644
--- a/crmd/lrm.c
+++ b/crmd/lrm.c
@@ -2255,7 +2255,7 @@ do_update_resource(const char *node_name, lrmd_rsc_info_t * rsc, lrmd_event_data
 
             if (remote_node) {
                 /* A container for a remote-node has started, initialize remote-node's status */
-                crm_info("Initalizing lrm status for container remote-node %s. Container successfully started.", remote_node);
+                crm_info("Initializing lrm status for container remote-node %s. Container successfully started.", remote_node);
                 remote_node_clear_status(remote_node, call_opt);
             } else if (container == FALSE && safe_str_eq(rsc->type, "remote") && safe_str_eq(rsc->provider, "pacemaker")) {
                 /* baremetal remote node connection resource has started, initialize remote-node's status */
