From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Wed, 10 Feb 2016 23:25:20 +0100
Subject: Fix typo: existance -> existence

---
 pengine/allocate.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pengine/allocate.c b/pengine/allocate.c
index 6462a0e..2e7687b 100644
--- a/pengine/allocate.c
+++ b/pengine/allocate.c
@@ -769,7 +769,7 @@ apply_system_health(pe_working_set_t * data_set)
         /* Requires the admin to configure the rsc_location constaints for
          * processing the stored health scores
          */
-        /* TODO: Check for the existance of appropriate node health constraints */
+        /* TODO: Check for the existence of appropriate node health constraints */
         return TRUE;
 
     } else {
@@ -985,7 +985,7 @@ sort_rsc_process_order(gconstpointer a, gconstpointer b, gpointer data)
     int r1_weight = -INFINITY;
     int r2_weight = -INFINITY;
 
-    const char *reason = "existance";
+    const char *reason = "existence";
 
     const GListPtr nodes = (GListPtr) data;
     resource_t *resource1 = (resource_t *) convert_const_pointer(a);
