From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Fri, 4 Mar 2016 10:34:10 +0100
Subject: Make the asciidoc documentation reproducible

---
 doc/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 9771293..2cf8f4e 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -72,7 +72,7 @@ endif
 EXTRA_DIST	= $(docbook:%=%.xml)
 
 %.html: %.txt
-	$(AM_V_ASCII)$(ASCIIDOC) --unsafe --backend=xhtml11 $<
+	$(AM_V_ASCII)$(ASCIIDOC) --unsafe --backend=xhtml11 --attribute footer-style=none $<
 
 # publican-clusterlabs/xsl/{html,html-single,pdf}.xsl refer to URIs
 # requiring Internet access, hence we shadow that with a XML catalog-based
